operations/Dockerfile

```diff
@@ -22,7 +22,7 @@ ARG TERRAFORM_VERSION
 # Install Terraform
 RUN yum install -y yum-utils
 RUN yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo
-RUN yum -y install terraform-${TERRAFORM_VERSION}-1
+RUN yum -y install terraform-${TERRAFORM_VERSION}
```

Comment:
Why the removal of the `-1`?

Reply:
I see further down it's added in the Dockerfile.

Reply:
yep, I wanted to have a single place that has the full version so that we don't have any "hey, why doesn't this do what I thought it did"-moments due to a split in version info